src/stravalib/model.py

```diff
@@ -460,19 +392,14 @@ class Shoe(Gear):
     pass
 
 
-class ActivityTotals(ActivityTotal, BackwardCompatibilityMixin):
+class ActivityTotals(ActivityTotal):
     """An objecting containing a set of total values for an activity including
     elapsed time, moving time, distance and elevation gain."""
 
-    _field_conversions = {
-        "distance": uh.meters,
-        "elevation_gain": uh.meters,
-    }
-    elapsed_time: Optional[timedelta] = None  # type: ignore[assignment]
-    moving_time: Optional[timedelta] = None  # type: ignore[assignment]
+    pass
```

Comment:
@jsamoocha as it was, we were converting distance and elevation gain to meters and time to timedelta type. but now we are no longer converting. Simpler code is great. However do we want to specify the units being returned in the docstring here for users and add a note about how values are potentially different?

For people who are maintaining  applications, will this significantly impact them?

Reply:
See my comment in #500. The impact should be minimal. The main breaking change will be if users are doing explicit instance checking such as `isinstance(obj_attribute, Quantity)`. That will not work anymore since we'll return a different type of object, but the behavior will remain the same.